job-server/src/spark.jobserver/JarManager.scala

```diff
@@ -5,9 +5,12 @@ import spark.jobserver.io.JobDAO
 import spark.jobserver.util.JarUtils
 import org.joda.time.DateTime
 
+import java.nio.file.{Files, Paths}
+
 // Messages to JarManager actor
 case class StoreJar(appName: String, jarBytes: Array[Byte])
 case object ListJars
+case class StoreInitialJars(initialJars: Map[String, String])
```

Comment:
Makes sense, however I wanted to differentiate the difference between the `StoreJar` that took the `jarBytes` and this message, which actually provides the path to be read. Do you think this deserves differentiation in naming? If not I can just change to `StoreJars`.

I'll also add some doc strings on the messages, thanks @noorul